eng/common/TestResources/Remove-TestResources.ps1

```diff
@@ -241,6 +257,9 @@ $verifyDeleteScript = {
 # Get any resources that can be purged after the resource group is deleted coerced into a collection even if empty.
 $purgeableResources = Get-PurgeableGroupResources $ResourceGroupName
 
+SetStorageNetworkAccessRules -ResourceGroupName $ResourceGroupName -AllowIpRanges $AllowIpRanges -Override -CI:$CI
```

Comment:
Why do we need to set these in order to clean-up?

Reply:
A pre-requisite for disabling WORM (which prevents storage account deletion) is deleting any blobs that are WORM enabled.

We cannot delete blobs if the network policy is set to `Deny` without any vnet/ip rules added, so we just get a 403 error from storage trying to delete the blob and fail out.